examples/fetch_costs_of_all_assemblies_in_timeframe.js

```diff
@@ -6,51 +6,49 @@
 //
 //   yarn prepack
 //
-const pMap = require('p-map')
-const { Transloadit } = require('transloadit')
+import pMap from 'p-map'
+import { Transloadit } from 'transloadit'
 
 const fromdate = '2020-12-31 15:30:00'
 const todate = '2020-12-31 15:30:01'
 
-;(async () => {
-  try {
-    const params = {
-      fromdate,
-      todate,
-      page: 1,
-    }
-
-    const transloadit = new Transloadit({
-      authKey: /** @type {string} */ (process.env.TRANSLOADIT_KEY),
-      authSecret: /** @type {string} */ (process.env.TRANSLOADIT_SECRET),
-    })
-
-    let totalBytes = 0
-
-    let lastCount
-    do {
-      console.log('Processing page', params.page)
-      const { count, items } = await transloadit.listAssemblies(params)
-      lastCount = count
-      params.page++
-
-      await pMap(
-        items,
-        // eslint-disable-next-line no-loop-func
-        async (assembly) => {
-          const assemblyFull = await transloadit.getAssembly(/** @type {string} */ (assembly.id))
-          // console.log(assemblyFull.assembly_id)
-
-          const { bytes_usage: bytesUsage } = assemblyFull
-
-          totalBytes += bytesUsage || 0
-        },
-        { concurrency: 20 }
-      )
-    } while (lastCount > 0)
-
-    console.log('Total GB:', (totalBytes / (1024 * 1024 * 1024)).toFixed(2))
-  } catch (err) {
-    console.error(err)
+try {
+  const params = {
+    fromdate,
+    todate,
+    page: 1,
   }
-})()
+
+  const transloadit = new Transloadit({
+    authKey: /** @type {string} */ (process.env.TRANSLOADIT_KEY),
+    authSecret: /** @type {string} */ (process.env.TRANSLOADIT_SECRET),
+  })
+
+  let totalBytes = 0
+
+  let lastCount
+  do {
+    console.log('Processing page', params.page)
+    const { count, items } = await transloadit.listAssemblies(params)
+    lastCount = count
+    params.page++
+
+    await pMap(
+      items,
+      // eslint-disable-next-line no-loop-func
+      async (assembly) => {
+        const assemblyFull = await transloadit.getAssembly(/** @type {string} */ (assembly.id))
+        // console.log(assemblyFull.assembly_id)
+
+        const { bytes_usage: bytesUsage } = assemblyFull
+
+        totalBytes += bytesUsage || 0
+      },
+      { concurrency: 20 }
+    )
+  } while (lastCount > 0)
+
+  console.log('Total GB:', (totalBytes / (1024 * 1024 * 1024)).toFixed(2))
+} catch (err) {
+  console.error(err)
+}
```

Comment:
All this catch block does is make sure it doesn’t affect the exit code. I don’t think this is desirable. I understand why this needed handling in CJS, but now we use top-level await. The try-catch can be removed. There are more occurrences of this.

This can also be done in a follow-up change.